src/doc/reference.md

```diff
@@ -2994,7 +2994,7 @@ Some examples of call expressions:
 # fn add(x: i32, y: i32) -> i32 { 0 }
 
 let x: i32 = add(1i32, 2i32);
-let pi: Option<f32> = "3.14".parse().ok();
+let pi: Option<f32> = "3.14".parse();
```

Comment:
Why not open a separate PR to update all the doc examples affected by #21718 instead of cramming it into this one?